File: Aula_02/Java/teste/src/Distancia_entre_dois_pontos.java
import java.util.Scanner;
import java.util.Locale;

public class Distancia_entre_dois_pontos {
    public static void main(String[] args){

        Locale.setDefault(Locale.US);

        Scanner sc = new Scanner(System.in);

        double x1 = sc.nextDouble();
        double y1 = sc.nextDouble();
        double x2 = sc.nextDouble();
        double y2 = sc.nextDouble();

        double valor1 = (x2 - x1) * (x2 - x1);
        double valor2 = (y2 - y1) * (y2 - y1);

        System.out.printf("%.4f-'",(Math.sqrt(valor1+valor2)));

        sc.close();
    }
}

File: Aula_02/C++/Distancia_entre_dois_pontos.cpp
#include <iostream>
#include <math.h>

int main(){
    double x1, y1, x2, y2;
    std::cin >> x1 >> y1;
    std::cin >> x2 >> y2;

    double valor1, valor2;
    valor1 = (x2 - x1) * (x2 - x1);
    valor2 = (y2 - y1) * (y2 - y1);

    std::cout << sqrt(valor1+valor2) << std::endl;

    return 0;
}